docs/stencil-docs/installing-stencil-cli/stencil-cli-options-and-commands.md

```diff
@@ -207,6 +207,8 @@ Usage: stencil push [<OPTIONS>]
 |`--save [<FILENAME]`           |`-s` | Specify the filename of the saved bundle                                          |
 |`--channel_id [<CHANNEL_ID>]`  |`-c` | Specify the channel ID of the storefront, if the store has multiple storefronts    |
 |`--activate [<VARIATIONNAME>]` |`-a` | Skip activation prompt; specify variation or leave blank to select first variation  |
+|                               |`-a -c 123 456`| Allows you to apply a theme to selected channels                           |
+|                               |`a -allc`| Allows you to apply a theme to all available channels.
```

Comment:
Full name of this option is `all_channels`, so user can input `--all_channels` instead of `-allc`. This option doesn't require any further specification (i.e. no channels ids required).